hips/hip-9999.md

```diff
@@ -0,0 +1,170 @@
+---
+hip: 9999
+title: "Helm 4 Development Process"
+authors: [ "Matt Butcher <matt.butcher@microsoft.com>" ]
+created: "2020-08-13"
```

Comment:
was this written back on August 13th or is this a typo? This is the same date as HIP 1.

Reply:
LOL... I wish I was that on top of things! Yeah, I will change it to the actual date.